docs/UserGuide.md

```diff
@@ -64,53 +64,65 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
+* When no optional fields are provided, e.g `clist`, the list will show all available details of all persons in the contact list.
+* When optional fields are provided, the list will only show the names of all persons in the contact list and the corresponding fields specified by the user.
+* More than one optional field can be provided.
+* The order of the optional fields does not matter. e.g both `clist e p` and `clist p e` will return a list of only the names, email addresses and phone numbers of all persons in the contact list.
+* If the specified field has no value for certain persons in the contact list, it will not show anything for that corresponding field for that particular person.
+
+Examples:
+* `clist` returns a list of all persons in the contact list with all the available details for each person.
+* `clist e p` returns a list of all persons in the contact list, showing only their names, email addresses and phone numbers (if available).
 
-Format: `list`
 
-### Editing a person : `edit`
+### Editing a person : `cedit`
 
-Edits an existing person in the address book.
+Edits an existing person in the contact list.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `cedit INDEX [n/NAME] [e/EMAIL] [p/PHONE] [a/ADDRESS] [th/TELEGRAM_HANDLE] [z/ZOOM] [dt/TAG_DELETED]… [t/TAG_ADDED]… `
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index must be a positive integer 1, 2, 3,…
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+* You can use `t/` to add a tag.
+* You can remove a specific tag by typing `dt/` with the tag.
+* You can remove all the person’s tags by typing `dt/*` without specifying any tags after it.
+* When editing tags, the tags to be deleted will be removed first, before new tags are added.
+
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `clist` followed by `cedit 2 p/91234567 e/agentX@thehightable.com` edits the phone number and email address of the 2nd person of the contact list to be 91234567 and agentX@thehightable.com respectively.
+* `cfind Betsy` followed by `cedit 2 n/Betsy Crower dt/*` edits the name of the 2nd person from the results of the cfind command to be Betsy Crower and clears all existing tags.
```

Comment:
* `cfind Betsy` followed by `cedit 2 n/Betsy Crower dt/*` edits the name of the 2nd person from the results of the `cfind` command to be `Betsy Crower` and clears all existing tags.